contracts/cw721-non-transferable/src/msg.rs

```diff
@@ -0,0 +1,97 @@
+use cosmwasm_schema::cw_serde;
+use cosmwasm_std::{Addr, Empty};
+use cw721_base::msg::QueryMsg as Cw721QueryMsg;
+
+#[cw_serde]
+pub struct InstantiateMsg {
+    pub admin: Option<String>,
+    pub name: String,
+    pub symbol: String,
+    pub minter: String,
+}
+
+#[cw_serde]
+pub enum QueryMsg {
+    Admin {},
+    OwnerOf {
+        token_id: String,
+        include_expired: Option<bool>,
+    },
+    Approval {
+        token_id: String,
+        spender: String,
+        include_expired: Option<bool>,
+    },
+    Approvals {
+        token_id: String,
+        include_expired: Option<bool>,
+    },
+    AllOperators {
+        owner: String,
+        include_expired: Option<bool>,
+        start_after: Option<String>,
+        limit: Option<u32>,
+    },
+    NumTokens {},
+    ContractInfo {},
+    NftInfo {
+        token_id: String,
+    },
+    AllNftInfo {
+        token_id: String,
+        include_expired: Option<bool>,
+    },
+    Tokens {
+        owner: String,
+        start_after: Option<String>,
+        limit: Option<u32>,
+    },
+    AllTokens {
+        start_after: Option<String>,
+        limit: Option<u32>,
+    },
+    Minter {},
+}
+
+impl From<QueryMsg> for Cw721QueryMsg<Empty> {
+    fn from(msg: QueryMsg) -> Cw721QueryMsg<Empty> {
+        match msg {
+            QueryMsg::OwnerOf {
+                token_id,
+                include_expired,
+            } => Cw721QueryMsg::OwnerOf {
+                token_id,
+                include_expired,
+            },
+            QueryMsg::NumTokens {} => Cw721QueryMsg::NumTokens {},
+            QueryMsg::ContractInfo {} => Cw721QueryMsg::ContractInfo {},
+            QueryMsg::NftInfo { token_id } => Cw721QueryMsg::NftInfo { token_id },
+            QueryMsg::AllNftInfo {
+                token_id,
+                include_expired,
+            } => Cw721QueryMsg::AllNftInfo {
+                token_id,
+                include_expired,
+            },
+            QueryMsg::Tokens {
+                owner,
+                start_after,
+                limit,
+            } => Cw721QueryMsg::Tokens {
+                owner,
+                start_after,
+                limit,
+            },
+            QueryMsg::AllTokens { start_after, limit } => {
+                Cw721QueryMsg::AllTokens { start_after, limit }
+            }
+            QueryMsg::Minter {} => Cw721QueryMsg::Minter {},
+            _ => unreachable!("cannot convert {:?} to Cw721QueryMsg", msg),
+        }
+    }
+}
+
+#[cw_serde]
+pub struct GetAdminResponse {
+    pub admin: Option<Addr>,
```

Comment:
Should be `Option<String>` in the response type as well

Reply:
done